### DISCUSSION TITLE
All audit event types

### DISCUSSION THREAD

**Discussion by rn3qlk** [gravitational/teleport#49503] [Q&A]
Hello,
Could you please help me identify all audit event types?

I have configured the transfer of audit events to my SIEM system using an event handler. I need to know all event types to map their fields correctly with my SIEM system.

In the future, these events will serve as a basis for configuring the correlation engine.

**Comment by stevenGravy** [comment]
The best place to look to for all event codes is here https://github.com/gravitational/teleport/blob/master/lib/events/codes.go

**Reply by rn3qlk** [reply]
Thank you so much!
